pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "Pymem"
+version = "1.9.0"
```

Comment:
I'm not familiar with how you do the release process in terms of tagging a commit, but it's possible to make this particular field dynamic so that you don't need to actually have the version hard-coded anywhere in the repo, and it is instead determined by using `importlib.metadata` which could be used since only 3.8+ is supported. Maybe it would be better as a separate ticket, but figured I'd mention it...

Reply:
> I would consider the version in pyproject.toml to be the single source of truth
> 
> it does seem I need to update it though and probably delete setup.py

fixed those, is that what you were talking about @monkeyman192?

Reply:
If you want to keep the version static, then yeah I think that's fine.
It might be good to add the `homepage`, `repository` and `documentation` fields as well since I think it will then make the pypi page cleaner with the links on the LHS?

Reply:
sounds like a good idea to me, what would the homepage be?

Reply:
I would have said homepage is what is on the LHS of pypi, but that is also the documentation link, so I think what you have added is fine?